pkg/smbios/type2_baseboard_information.go

```diff
@@ -33,7 +33,7 @@ func ParseBaseboardInfo(t *Table) (*BaseboardInfo, error) {
 	if t.Type != TableTypeBaseboardInfo {
 		return nil, fmt.Errorf("invalid table type %d", t.Type)
 	}
-	if t.Len() < 0xf {
+	if t.Len() < 0x8 {
```

Comment:
what is this number?

Reply:
It's the minimum length of type 2 structure,
defined in DSP0134 7.3, length of the structure is at least 08h. The original code would return early if the length is smaller than 0xf which should be incorrect.

Reply:
can you put a comment to that effect here? People need to understand why there is an 8.

Reply:
Comment added.

Reply:
Hi @insomniacslk @pmazzini, appreciate any review comment.
Thanks.